vspace/src/main/java/edu/asu/diging/vspace/web/exhibit/publicview/ExhibitionModuleController.java

```diff
@@ -29,6 +34,9 @@ public String module(@PathVariable("id") String id, @PathVariable("spaceId") Str
         if (space == null) {
             return "redirect:/exhibit/404";
         }
+        ExhibitionAboutPage exhibitionAboutPage = aboutPageManager.getExhibitionAboutPage();
+        model.addAttribute("aboutPageTitle", exhibitionAboutPage.getTitle());
+        model.addAttribute("aboutPageText", exhibitionAboutPage.getAboutPageText()); 
```

Comment:
is that needed here? about page is its own page, isn't it?

Reply:
This is to allow the staff to edit the about page on the staff site, so i am taking the about page content and providing it here to allow them to edit. Is this okay?

Reply:
On the exhibition pages(module, space and slides) the navbar on top will only show the about page button if it is setup by the staff. Line 37-39 is there to check th:if if they exist in order to show the button. Is this okay? 
<img width="739" alt="image" src="https://user-images.githubusercontent.com/22446190/155763299-d7ca4822-6d64-4392-b995-871d2537b14b.png">